components/treeview/metadata/treeview.yml

```diff
@@ -77,117 +77,383 @@ examples:
         <li class="spectrum-TreeView-item">
-          <a class="spectrum-TreeView-itemLink" href="#">
-            <svg class="spectrum-Icon spectrum-Icon--sizeS" focusable="false" aria-hidden="true" aria-label="Layers">
-              <use xlink:href="#spectrum-icon-18-Layers"></use>
-            </svg>
-            Layer 1
-          </a>
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 1</span></a>
+        </li>
+
+        <li class="spectrum-TreeView-item is-selected">
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 2</span></a>
         </li>
+      </ul>
 
+  - id: treeview
+    name: Standalone
+    description: |
+      A treeview not nested within a panel.
+    markup: |
+      <ul class="spectrum-TreeView spectrum-TreeView--standalone" style="width: 250px">
+        <li class="spectrum-TreeView-item">
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 1</span></a>
+        </li>
+
+        <li class="spectrum-TreeView-item is-selected">
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 2</span></a>
+        </li>
+      </ul>
+
+  - id: treeview
+    name: Quiet
+    description: |
+      A treeview with quiet selection.
+    markup: |
+      <ul class="spectrum-TreeView spectrum-TreeView--quiet" style="width: 250px">
+        <li class="spectrum-TreeView-item">
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 1</span></a>
+        </li>
+
+        <li class="spectrum-TreeView-item is-selected">
+          <a class="spectrum-TreeView-itemLink" href="#"><span class="spectrum-TreeView-itemLabel">Layer 2</span></a>
+        </li>
+      </ul>
+
+  - id: treeview
+    name: Folders & Files
+    description: A nested treeview with folders and files.
+    markup: |
+      <ul class="spectrum-TreeView" style="width: 250px">
         <li class="spectrum-TreeView-item is-open">
           <a class="spectrum-TreeView-itemLink" href="#">
-            <svg class="spectrum-Icon spectrum-UIIcon-ChevronRightMedium spectrum-TreeView-indicator" focusable="false" aria-hidden="true">
+            <svg class="spectrum-Icon spectrum-UIIcon-ChevronRightMedium spectrum-TreeView-itemIndicator" focusable="false" aria-hidden="true">
               <use xlink:href="#spectrum-css-icon-ChevronRightMedium" />
             </svg>
-            <svg class="spectrum-Icon spectrum-Icon--sizeS" focusable="false" aria-hidden="true" aria-label="Folder">
-              <use xlink:href="#spectrum-icon-18-Folder"></use>
+            <svg class="spectrum-TreeView-itemIcon spectrum-Icon spectrum-Icon--sizeS" focusable="false" aria-hidden="true" aria-label="Folder">
+              <use xlink:href="#spectrum-icon-18-Folder"><span class="spectrum-TreeView-itemLabel"></span></use>
```

Comment:
I think this `<span />` should not inside `<use />` and it is redundant. I will remove it along with the change to add thumbnail component VRT scenario.

Reply:
Ah, looks like I screwed up the regex when I added the label. Thanks!